packages/manager/apps/vrack-services/src/components/listing/VrackAssociationModal.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+// import { OsdsModal } from '@ovhcloud/ods-components/modal/react';
+import {
+  ODS_BUTTON_TYPE,
+  ODS_BUTTON_VARIANT,
+} from '@ovhcloud/ods-components/button';
+import { OsdsButton } from '@ovhcloud/ods-components/button/react';
+import { ODS_MESSAGE_TYPE } from '@ovhcloud/ods-components/message';
+import { OsdsMessage } from '@ovhcloud/ods-components/message/react';
+import { OsdsSpinner } from '@ovhcloud/ods-components/spinner/react';
+import { ODS_SPINNER_SIZE } from '@ovhcloud/ods-components/spinner';
+import { useVrackServicesAllowedVrack } from '@/api';
+import { AssociateVrack } from './AssociateVrack';
+import { CreateVrack } from './CreateVrack';
+
+export type VrackAssociationModalProps = {
+  vrackServicesId?: string;
+  disabled?: boolean;
+  closeModal: () => void;
+};
+
+export const VrackAssociationModal: React.FC<VrackAssociationModalProps> = ({
+  vrackServicesId,
+  closeModal,
+}) => {
+  const { t } = useTranslation('vrack-services/listing');
+  const { allowedVrackList, isError, isLoading } = useVrackServicesAllowedVrack(
+    vrackServicesId,
+  );
+
+  if (!vrackServicesId) {
+    return <></>;
+  }
+
+  return (
+    // TODO: Put back ODS modal when the buttons work
+    // <OsdsModal
+    //   dismissible
+    //   headline={t('modalVrackAssociationTitle')}
+    //   masked={!vrackServicesId || undefined}
+    //   onOdsModalClose={closeModal}
+    // >
+    <div
+      style={{
+        display: vrackServicesId ? 'flex' : 'none',
+        flexDirection: 'column',
+        padding: '20px',
+        margin: '20px',
+        border: '3px solid black',
+      }}
```

Comment:
Because it's temporary while we don't have a modal
Why would you create an additional file to remove it when the modal is ready ?
Do the fastest thing

And you risk to forget the css file once you replace the modal